programs/loader-v4/src/lib.rs

```diff
@@ -1589,7 +1589,7 @@ mod tests {
             &[0, 1, 2, 3],
             transaction_accounts.clone(),
             &[(1, false, true)],
-            Err(InstructionError::InvalidAccountOwner),
+            Err(InstructionError::UnsupportedProgramId),
```

Comment:
Loader-v4 is not scheduled for activation on any network, so changes have no effect.